### PR TITLE
Add Chain State Component

### LINKED ISSUE #1: Chain State Component
We should create a component equivalent to the "Chain State" tab in the Polkadot JS Apps

### PR DESCRIPTION
**Pull request by shawntabrizi**
Pretty broken as is. It works when nothing is wrong, but crashes if any error occurs.

Closes #1

### CODE REVIEW

File: src/ChainState.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      text: module
+    }));
+
+    setModulesList(modules);
+  }, [api]);
+
+  useEffect(() => {
+    if (module !== "") {
+      let queries = eval("api.query." + module + ".valueOf()");
+      let storageItems = Object.keys(queries).map(storage => ({
```
Comment (jacogr) [inline src/ChainState.jsx]: Can just be `Object.keys(api.query[module])` (no eval)

File: src/ChainState.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      return {
+        ...formState,
+        [data.state]: data.value
+      };
+    });
+  };
+
+  const runQuery = async () => {
+    let result = await eval(
+      "api.query." + module + "." + storageItem + "('" + input + "')"
```
Comment (jacogr) [inline src/ChainState.jsx]: Can just be `await api.query[module][storageItem](input)` - no eval

File: src/ChainState.jsx (hunk 1 above)
Comment (shawntabrizi) [reply src/ChainState.jsx]: perfect, thank you

File: src/ChainState.jsx (hunk 2 above)
Comment (shawntabrizi) [reply src/ChainState.jsx]: Is there an easy way to learn if the query needs an input?

i.e. free balance expects a user account, while sudo key needs no input

**Pull request merged**